src/assets/main.css

```
@@ -22,95 +60,93 @@
+  @apply text-base font-medium leading-p-150 -tracking-[0.02em];
+  font-feature-settings: 'ss15' on;
+}
+
+.text8 {
+  @apply text-s-17 font-normal leading-p-150;
+}
+
+/**
+* min screen-tablet-portain
+*/
+@media (width >=theme(--breakpoint-md)) {
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)